compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -541,6 +541,20 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_attribute_with_speed_marks() {
+        let input = r#"#[test(should_fail_with = "message")]"#;
```

Comment:
This test now doesn't test for the original case anymore which used an apostrophe